packages/select/src/vaadin-select.js

```diff
@@ -641,6 +688,30 @@ class Select extends DelegateFocusMixin(FieldMixin(SlotMixin(ElementMixin(Themab
     return !(this.invalid = !(this.disabled || !this.required || this.value));
   }
 
+  /**
+   * Renders items when they are provided by the `items` property and clears the content otherwise.
+   * @param {HTMLElement} root
+   * @param {Select} _select
+   * @private
+   */
+  __defaultRenderer(root, _select) {
+    if (this.items.length === 0) {
+      root.textContent = '';
+      return;
+    }
+
+    let listBox = root.firstElementChild;
+    if (!listBox) {
+      listBox = document.createElement('vaadin-list-box');
```

Comment:
Should we use `vaadin-select-list-box` to align with what `vaadin-context-menu` does?

Reply:
Done.